docs/users.rst

```diff
@@ -674,8 +674,11 @@ ref
 
 pkg_managers
   A list of package managers to be used for resolving the upstream project
-  dependencies. If not provided, cachito will try to figure out what manager to
-  use here.
+  dependencies. If not provided, cachito will assume ``gomod`` due to backward
```

Comment:
Please keep in mind that the default package manager is deployment specific, so I'm not sure if that should be included in the upstream documentation.

Reply:
Doesn't Cachito set that as `gomod` for all envs by default? Having cachito not to set `gomod` as the value for DEFAULT_PACKAGE_MANAGERS is an opt-out feature and not opt-in, right?

Any suggestions on how to proper phrase this here? Moreover, do you think we should make this a required param in OSBS at some point?

I do feel that omitting this information could do more harm than good.

Finally, if we do not have it in place yet, it would be nice to throw a warning to let users know they should explicitly use the pkg_managers key.

Reply:
> Doesn't Cachito set that as `gomod` for all envs by default? Having cachito not to set `gomod` as the value for DEFAULT_PACKAGE_MANAGERS is an opt-out feature and not opt-in, right?

Yes, that is true.

> Any suggestions on how to proper phrase this here? Moreover, do you think we should make this a required param in OSBS at some point?
> 
> I do feel that omitting this information could do more harm than good.

How about after the end of the sentence, you add the following:
```
... compatibility reasons, however, this default could be configured differently on different Cachito deployments.
```
I like the idea of making it a requirement but it would involve us hunting down all getting existing Cachito users to have them change their `container.yaml` file, so it may not be worth the effort.

> 
> Finally, if we do not have it in place yet, it would be nice to throw a warning to let users know they should explicitly use the pkg_managers key.

Reply:
Fixed. I also made sure to let the user know they should check it with the cachito instance adimins.

Reply:
Looks good. The only thing that bugs me is that some places use `cachito` instead of `Cachito`.